docs/dev-notes.md

```
@@ -15,7 +14,10 @@
 - consider using drawer for contact button
 - consider using drawer for donate button
 - use drawer component in mobile view and dialog in desktop
-- consider new button variant for footer buttons to change outline after small breakpoint
 - consider making logo icon bigger
-- change sign-in button colors
 - look into how easy it is to use stripe in a small form component in drawer.
+- consider adding drizzle-kit push --force to preview branches:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)